main.py

```diff
@@ -94,6 +94,7 @@ class App:
     def __init__(self, name: str, repo: str):
         self.name = name
         self.repo = repo
+        self.link = repo[:-4] if repo.endswith(".git") else repo
```

Comment:
Python 3.9 offers a cleaner way: https://docs.python.org/3.9/library/stdtypes.html?highlight=removeprefix#str.removesuffix

Reply:
We can update it anywhere I guess.

Don't really care about backward compatibility given it's a very fire-and-forget script.

Reply:
> Don't really care about backward compatibility given it's a very fire-and-forget script.

Fair. I'm used to supporting >= 3.8 for my Python tools.

And done.